definition/http.go

```diff
@@ -20,5 +20,22 @@ type Request struct {
 type Response struct {
 	StatusCode int `json:"statusCode"`
 	headers
-	Body string `json:"body"`
+	Persisted  Persisted `json:"persisted"`
+	Body       string    `json:"body"`
+	BodyAppend string    `json:"BodyAppend"`
```

Comment:
You might not want to use the persisted functionality, but just return the request body and append some JSONs fields to it. Actually if there's persisted node in the request the response will be loaded from there, but not from response body and bodyAppend.